modules/heat_transfer/doc/content/source/kernels/HeatSource.md

```diff
@@ -1 +1,31 @@
-!template load file=stubs/moose_object.md.template name=HeatSource syntax=/Kernels/HeatSource
\ No newline at end of file
+# HeatSource
+
+## Description
+
+The `HeatSource` kernel implements a volumetric heat source/sink forcing term. The strong form is
+
+\begin{equation}
+\underbrace{-\dot{q}}_{\textrm{HeatSource}} + \text{other kernels} = 0 \in \Omega
+\end{equation}
+
+where $\dot{q}$ is the volumetric heat source. The corresponding weak form,
+in inner-product notation, is
+
+\begin{equation}
+R_i(u_h)=(\psi_i, -\dot{q})\quad\forall \psi_i,
+\end{equation}
+
+where $u_h$ is the approximate solution and $\psi_i$ is a finite element test function.
+
+A slightly more general version of this same kernel can be found in [BodyForce](BodyForce.md), which you can equivalently use instead of `HeatSource`.
```

Comment:
```suggestion
A slightly more general version of this same kernel can be found in [BodyForce](BodyForce.md), which you can equivalently used instead of `HeatSource`.
```

Reply:
I think the original version is correct grammar

Reply:
lol for sure. I think I meant to edit some other grammar and then made this silliness instead